### DISCUSSION TITLE
Upgrading  React-Calendar SPFx version to 1.13.0 Version

### DISCUSSION THREAD

**Discussion by mohammadamer** [pnp/sp-dev-fx-webparts#2130] [General]
**Upgrading React-Calendar SPFx version to 1.13.0 version**
I am trying to upgrade React-Calendar SPFx version to 1.13.0 version.
Although the build went successfully, I got an issue with bundling the solution.  gulp bundle --ship
Here is the details:

```
Build target: SHIP
Build target: SHIP
[14:31:44] Using gulpfile ~\source\repos\CalendarWebPart\gulpfile.js
[14:31:44] Starting 'bundle'...
[14:31:44] Starting gulp
[14:31:44] Starting gulp
[14:31:44] Starting subtask 'pre-copy'...
[14:31:45] Finished subtask 'pre-copy' after 862 ms
[14:31:45] Starting subtask 'stylelint'...
[14:31:50] Finished subtask 'stylelint' after 5.76 s
[14:31:50] Starting subtask 'copy-static-assets'...
[14:31:50] Starting subtask 'sass'...
[14:31:51] Warning - [sass] src\webparts\calendar\components\calendar.scss: filename should end with module.sass or module.scss
[14:31:53] Finished subtask 'sass' after 3.08 s
[14:31:53] Starting subtask 'tslint'...
[14:31:54] [tslint] tslint version: 5.20.1
[14:31:54] Starting subtask 'tsc'...
[14:31:54] [tsc] typescript version: 3.9.10
[14:31:55] Finished subtask 'copy-static-assets' after 4.13 s
[14:32:22] Finished subtask 'tslint' after 28 s
[14:32:25] Finished subtask 'tsc' after 31 s
[14:32:25] Starting subtask 'post-copy'...
[14:32:25] Finished subtask 'post-copy' after 535 μs
[14:32:25] Starting subtask 'configure-webpack'...
[14:32:28] Finished subtask 'configure-webpack' after 3.45 s
[14:32:28] Starting subtask 'webpack'...
[14:34:38] Warning - [webpack] 'dist':
./node_modules/fbjs/lib/setImmediate.js
There are multiple modules with names that only differ in casing.
This can lead to unexpected behavior when compiling on a filesystem with other case-semantic.
Use equal casing. Compare these module identifiers:
* {project-path}\CalendarWebPart\node_modules\fbjs\lib\setImmediate.js
    Used by 1 module(s), i. e.
    {project-path}\CalendarWebPart\node_modules\draft-js\lib\editOnBeforeInput.js
* {project-path}\CalendarWebPart\node_modules\fbjs\lib\setimmediate.js
    Used by 2 module(s), i. e.
    {project-path}\CalendarWebPart\node_modules\fbjs\lib\setImmediate.js
 @ ./node_modules/fbjs/lib/setImmediate.js
 @ ./node_modules/draft-js/lib/editOnBeforeInput.js
 @ ./node_modules/draft-js/lib/DraftEditorEditHandler.js
 @ ./node_modules/draft-js/lib/DraftEditor.react.js
 @ ./node_modules/draft-js/lib/Draft.js
 @ ./lib/controls/Event/event.js
 @ ./lib/webparts/calendar/components/Calendar.js
 @ ./lib/webparts/calendar/CalendarWebPart.js

[14:34:38] Finished subtask 'webpack' after 2.15 min
[14:34:38] Finished 'bundle' after 2.9 min
[14:34:38] ==================[ Finished ]==================
Warning - [sass] src/webparts/calendar/components/calendar.scss: filename should end with module.sass or module.scss
Warning - [webpack] 'dist':
./node_modules/fbjs/lib/setImmediate.js
There are multiple modules with names that only differ in casing.
This can lead to unexpected behavior when compiling on a filesystem with other case-semantic.
Use equal casing. Compare these module identifiers:
* {project-path}/CalendarWebPart/node_modules/fbjs/lib/setImmediate.js
    Used by 1 module(s), i. e.
    {project-path}/CalendarWebPart/node_modules/draft-js/lib/editOnBeforeInput.js
* {project-path}/CalendarWebPart/node_modules/fbjs/lib/setimmediate.js
    Used by 2 module(s), i. e.
    {project-path}/CalendarWebPart/node_modules/fbjs/lib/setImmediate.js
 @ ./node_modules/fbjs/lib/setImmediate.js
 @ ./node_modules/draft-js/lib/editOnBeforeInput.js
 @ ./node_modules/draft-js/lib/DraftEditorEditHandler.js
 @ ./node_modules/draft-js/lib/DraftEditor.react.js
 @ ./node_modules/draft-js/lib/Draft.js
 @ ./lib/controls/Event/event.js
 @ ./lib/webparts/calendar/components/Calendar.js
 @ ./lib/webparts/calendar/CalendarWebPart.js

[14:34:39] Project react-calendar version:1.0.4
[14:34:39] Build tools version:3.17.17
[14:34:39] Node version:v14.17.0
[14:34:39] Total duration:3.5 min
[14:34:39] Task warnings:2
The build failed because a task wrote output to stderr.
Exiting with exit code: 1
```

**My findings**
I see that there is an old similar issue [https://github.com/pnp/sp-dev-fx-webparts/issues/1181](url)
I tried to run spfx doctor but seems it's not supported with SPFx 1.13.0
`Error: spfx doctor doesn't support SPFx v1.13.0 at this moment`

Also I have noticed that a similar issue reported about fbjs [https://github.com/facebook/fbjs/issues/234](url) 
@hugoabernier, @joaojmendes, @Abderahman88 
Could you please help on that or give some hints how to deep dive investigating the issue ?

**Another Issue With Bundling in another solution**
I have another issue also with bundling but with another solution after Upgrading to SPFx 1.13.0
**[webpack] TypeError: Cannot read property 'get' of undefined
webpack' sub task errored after 1.18 s 
Cannot read property 'get' of undefined**

-  Build target: SHIP
-  [14:06:31] Using gulpfile ~\source\repos\PushNotificationsFrontend\gulpfile.js
-  [14:06:31] Starting 'bundle'...
-  [14:06:31] Starting gulp
-  [14:06:31] Starting subtask 'pre-copy'...
-  [14:06:32] Finished subtask 'pre-copy' after 1.1 s
-  [14:06:32] Starting subtask 'copy-static-assets'...
-  [14:06:32] Starting subtask 'sass'...
-  [14:06:32] Finished subtask 'sass' after 90 ms
-  [14:06:32] Starting subtask 'tslint'...
-  [14:06:33] [tslint] tslint version: 5.20.1
-  [14:06:34] Starting subtask 'tsc'...
-  [14:06:34] [tsc] typescript version: 3.9.10
-  [14:06:35] Finished subtask 'copy-static-assets' after 2.24 s
-  [14:06:43] Finished subtask 'tsc' after 9.06 s
-  [14:06:43] Finished subtask 'tslint' after 10 s
-  [14:06:43] Starting subtask 'post-copy'...
-  [14:06:43] Finished subtask 'post-copy' after 751 μs
-  [14:06:43] Starting subtask 'configure-webpack'...
-  [14:06:46] Finished subtask 'configure-webpack' after 2.78 s
-  [14:06:46] Starting subtask 'webpack'...
-  [14:06:47] Error - [webpack] TypeError: Cannot read property 'get' of undefined
-  [14:06:47] Error - 'webpack' sub task errored after 1.18 s 
-   Cannot read property 'get' of undefined
-  [14:06:47] 'bundle' errored after 16 s
-  [14:06:47]
-  About to exit with code: 1
-  Process terminated before summary could be written, possible error in async code not continuing!
-  Trying to exit with exit code 1

**Comment by donrandika** [comment]
I'm also getting this error. Is there anyone know how to fix this?

**Reply by mohammadamer** [reply]
Hi @donrandika 

I think these are warning not errors so for my case I ignored them and packaged the solution.

**Reply by donrandika** [reply]
Yes, these are warning messages, but I ignored them and proceeded with building it. Thanks.
